### DISCUSSION TITLE
您好，咨询mmpose关于物体关键点检测的一些问题

### DISCUSSION THREAD

**Discussion by ShaneCan** [open-mmlab/mmpose#1109] [Ideas] [answered]
mmpose团队的各位大神好，本人有以下几个问题请教。
1.请问是否可以使用mmpose来进行飞机的关键点检测呢？
2.如果可以的话，在制作数据集时应采用2D Body Keypoint模型或2D Animal Keypoint模型或其他模型？
3.mmpose关键点数据集应当如何制作呢？是否需要加掩膜？对于被遮挡的点如何标注呢？

衷心感谢mmpose制作团队的回答，期望mmpose越来越好。

**Comment by jin-s13** [comment, marked as answer]
1.我认为可以用mmpose来检测飞机的关键点。
2. 实际上 2D Body Keypoint 和 2D Animal Keyponit 的网络结构都是通用的，只是数据集不同。2d_kpt_sview_rgb_img里面的模型应该都是可以用的。
3. （1）数据集制作的话，可以试一下 https://github.com/open-mmlab/labelbee-client 工具。但是这个工具目前，无法直接转换成coco格式，还需要手动转换一下格式。（2）不需要mask信息 （3）被遮挡点一种方式是正常脑补坐标[x, y, 1]，一种方式是标注为[0,0,0]。两种方式均可。
